Makefile

```diff
@@ -338,10 +338,11 @@ foss-checks:
 ###############################################################################
 .PHONY: ci
 ## Run what CI runs
-ci: clean format-check images test $(BINDIR)/gen-versions validate-gen-versions dirty-check
+ci: clean format-check images test dirty-check validate-gen-versions
```

Comment:
It seems a little redundant to have `dirty-check` here and also inside `validate-gen-versions` but I don't know enough about makefile execution order to suggest a working alternative so lgtm

Reply:
Yeah it is certainly redundant in the ci target but I thought it would be better to dirty-check specifically in validate-gen-versions so it would be obvious that the dirty was due to the gen-versions. It also allows the validate-gen-versions target to be a standalone check so if ci was changed to remove the dirty-check the validate-gen-versions would still catch a problem.